tests/admin.test.ts

```diff
@@ -182,3 +184,105 @@ describe('bonus points submission', () => {
     expect(getNoBonusUserResponse.user.points).toEqual(0);
   });
 });
+
+describe('event creation', () => {
+  test('successful event creation', async () => {
```

Comment:
nit: add in some comments that divide the testing into sections like
// setting up inputs
// verify response from event creation
// verify response from database lookup